index.html

```diff
@@ -36,134 +38,127 @@
+              title="Berlin's finest JavaScript Usergroup">meetup.com groups.</a>. 
+              Especially <a href="http://berlinjs.org/" title="Berlin's finest JavaScript Usergroup">berlinjs.org</a>
+              is a great place to start.
+          </p>
         </div>
-        <div>
-          <h2>The team</h2>
+        <div class="speaker">
+          <h2 class="header-style">Hosting an event?</h2>
           <p>
-            ReactJS.berlin is organized by
-            <a href="http://www.twitter.com/chris_bitcrowd" title="Follow Chris on Twitter">Christoph Beck</a>
-            from
-            <a href="http://www.twitter.com/bitcrowd" title="Follow bitcrowd on Twitter">@bitcrowd</a>. Interested in joining the organizers team? Drop us a line on twitter or meetup :)
-            Design by <a href="http://www.twitter.com/m_besser" title="Follow Matti on Twitter">Matti Besser</a> &amp; <a href="http://www.twitter.com/plankton" title="Follow Darren on Twitter">Darren Cadwallader</a>.
+            The <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd</a> office has an event space that you
+            can use for community events. It has a capacity of 90 people (seated) or 30 people (with desks for courses).
           </p>
         </div>
-        <div>
-          <h2>Supporters</h2>
+        <div class="speaker">
+          <h2 class="header-style">Meet our React experts</h2>
           <p>
-            Our Meetups are hosted by <a href="http://bitcrowd.net">bitcrowd.net</a>.
-            The template for this page is the gloriuos <a href="http://berlinjs.org/" title="Berlin's finest JavaScript Usergroup">berlinjs.org page</a> with kind permission of <a href="http://www.twitter.com/rmehner" title="Follow Robin on Twitter">Robin Mehner</a>.
+            We at <a href="http://bitcrowd.net" title="bitcrowd.net office">@bitcrowd</a> have created many ReactJS applications 
+            for our clients over the past few years. Just drop us an email or give us a call and we’ll see how we can help 
+            your project.
           </p>
         </div>
       </div>
-      <section>
+      <section class="clearfix three-cols">
+          <div itemscope="" itemtype="http://schema.org/Organization">
+            <h3>Get in touch</h3>
+            <ul>
+              <li itemprop="email">
+                <a href="mailto:info@bitcrowd.net?Subject=Hallo bitcrowd!">
+                  <span>
+                   Email: info@bitcrowd.net
+                  </span>
+                </a>
+              </li>
+              <li itemprop="twitter">
+                <a href="https://twitter.com/bitcrowd">
+                  <span>
+                    Twitter: @bitcrowd
+                  </span>
+                </a>
+              </li>
+              <li itemprop="telephone">
+                <a href="tel:493055577597">
+                  <span>
+                    +49 30 555 77 597
```

Comment:
If we say `email:` & `twitter:`, we should say `tel:` here, or remove all of them I think

Reply:
Okay, I removed the email & twitter part. 👍

Reply:
Great : 👍